components/hacbs/jvm-build-service/base/self-binding.yaml

```diff
@@ -8,12 +8,15 @@ spec:
     workspace:
       exportName: jvm-build-service
   permissionClaims:
-  - group: ""
-    resource: secrets
-    state: Accepted
-  - group: ""
-    resource: serviceaccounts
-    state: Accepted
-  - group: ""
-    resource: events
-    state: Accepted
+    - group: ""
```

Comment:
I'm also going to try with the entire permissionClaims section here removed, where we just merge with what's in the jvm-build-service repo.

I will report back / update if that works

Reply:
sure enough this form works and should prove to be the more stable long term, as we can manage the permission claims for the binding solely from the jvm-build-service repo